tracpro/polls/tasks.py

```diff
@@ -92,7 +92,9 @@ def fetch_org_updated_runs(org):
     incomplete_responses = Response.get_update_required(org)
 
     if incomplete_responses:
-        runs = client.get_runs(ids=[r.flow_run_id for r in incomplete_responses])
+        runs = []
+        for i in range(0, len(incomplete_responses), 15):
```

Comment:
Since we do these sort of batching operations quite a bit Im going to propose this as a re-usable solution: https://github.com/rapidpro/dash/pull/19. In this case you'd write...

```python
runs = []
for resp_chunk in chunks(list(incomplete_responses), 50):
  runs += client.get_runs(ids=[r.flow_run_id for r in resp_chunk])
```